frappe/public/js/frappe/form/grid_row.js

```diff
@@ -126,6 +126,8 @@ export default class GridRow {
 						console.trace(e);
 					});
 			} else {
+				this.hide_form();
+
```

Comment:
Maybe move lines 99:101 outside the `if` block?

Reply:
Yep it's better indeed.